src/TorchSharp/TorchVision/models/AlexNet.cs

```diff
@@ -81,14 +83,19 @@ public AlexNet(int numClasses, float dropout = 0.5f, torch.Device device = null)
                     Dropout(probability: dropout),
                     Linear(256 * 6 * 6, 4096),
                     ReLU(inPlace: true),
-                    Dropout(probability:dropout),
+                    Dropout(probability: dropout),
                     Linear(4096, 4096),
                     ReLU(inPlace: true),
                     Linear(4096, numClasses)
                 );
 
                 RegisterComponents();
 
+                if (!string.IsNullOrEmpty(weights_file)) {
+
+                    this.load(weights_file, skip: skipfc ? new[] { "classifier.6.weight", "classifier.6.bias" } : null);
```

Comment:
> new[] { "classifier.6.weight", "classifier.6.bias" } [](http://example.com/codeflow?start=58&length=53)

Is this a mutable array? I am asking to see if we can have it as static and avoid the allocations?